CITATIONS.md

```diff
@@ -0,0 +1,258 @@
+| [Self-report captures 27 distinct categories of emotion bridged by continuous gradients](https://doi.org/10.1073/pnas.1702247114)                                                    | 2017 |  multi   | [Cite](#1)  |
+| [Mapping the Passions: Toward a High-Dimensional Taxonomy of Emotional Experience and Expression](https://doi.org/10.1177/1529100619850176)                                          | 2019 |  multi   | [Cite](#2)  |
+| [The primacy of categories in the recognition of 12 emotions in speech prosody across two cultures](https://doi.org/10.1038/s41562-019-0533-6)                                       | 2019 |  voice   | [Cite](#3)  |
+| [Mapping 24 emotions conveyed by brief human vocalization](https://doi.org/10.1037/amp0000399)                                                                                       | 2019 |  voice   | [Cite](#4)  |
+| [Emotional expression: Advances in basic emotion theory](https://doi.org/10.1007%2Fs10919-019-00293-3)                                                                               | 2019 |  multi   | [Cite](#5)  |
+| [What the face displays: Mapping 28 emotions conveyed by naturalistic expression](https://doi.org/10.1037/amp0000488)                                                                | 2020 |   face   | [Cite](#6)  |
+| [The neural representation of visually evoked emotion is high-dimensional, categorical, and distributed across transmodal brain regions](https://doi.org/10.1016/j.isci.2020.101060) | 2020 |  multi   | [Cite](#7)  |
+| [What music makes us feel: At least 13 dimensions organize subjective experiences associated with music across different cultures](https://doi.org/10.1073/pnas.1910704117)          | 2020 |  music   | [Cite](#8)  |
+| [GoEmotions: A Dataset of Fine-Grained Emotions](https://doi.org/10.18653/v1/2020.acl-main.372)                                                                                      | 2020 |   text   | [Cite](#9)  |
+| [Universal facial expressions uncovered in art of the ancient Americas: A computational approach](https://doi.org/10.1126/sciadv.abb1005)                                            | 2020 |   face   | [Cite](#10) |
+| [Sixteen facial expressions occur in similar contexts worldwide](https://doi.org/10.1038/s41586-020-3037-7)                                                                          | 2021 |   face   | [Cite](#11) |
+| [The MuSe 2022 Multimodal Sentiment Analysis Challenge: Humor, Emotional Reactions, and Stress](https://doi.org/10.48550/arXiv.2207.05691)                                           | 2022 |  multi   | [Cite](#12) |
+| [The ACII 2022 Affective Vocal Bursts Workshop & Competition: Understanding a critically understudied modality of emotional expression](https://doi.org/10.48550/arXiv.2207.03572)   | 2022 |  voice   | [Cite](#13) |
```

Comment:
note for self really, need to remember to update this with the acii proceedings bib when we have it